src/mlstacks/constants.py

```diff
@@ -11,6 +11,29 @@
 #  or implied. See the License for the specific language governing
 #  permissions and limitations under the License.
 """MLStacks constants."""
+import os
+
+
+def handle_bool_env_var(var: str, default: bool = False) -> bool:
+    """Converts normal env var to boolean.
+
+    Args:
+        var: The environment variable to convert.
+        default: The default value to return if the env var is not set.
+
+    Returns:
+        The converted value.
+    """
+    value = os.getenv(var)
+
+    if value in ["1", "y", "yes", "True", "true"]:
+        return True
+
+    if value in ["0", "n", "no", "False", "false"]:
+        return False
+
+    return default
```

Comment:
Should this be a util function instead of living in constants.py?

Reply:
I copied the behavior from the main ZenML repo. If you think that it is more convenient, we can move it to a file like: `utils/environment_utils.py`.

Reply:
Yes I think that'd be better.

Reply:
On it!